commons/all/src/main/java/org/infinispan/commons/configuration/attributes/Attribute.java

```diff
@@ -93,6 +97,10 @@ public boolean isModified() {
       return modified;
    }
 
+   public boolean isRepeated() {
```

Comment:
unable to find any usage. are you planning to use it in the future?

Reply:
Yes